src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionInfo.cs

```diff
@@ -39,17 +41,20 @@ public ConnectionInfo(ISqlConnectionFactory factory, string ownerUri, Connection
         /// <summary>
         /// Factory used for creating the SQL connection associated with the connection info.
         /// </summary>
-        public ISqlConnectionFactory Factory {get; private set;}
+        public ISqlConnectionFactory Factory { get; private set; }
 
         /// <summary>
         /// Properties used for creating/opening the SQL connection.
         /// </summary>
         public ConnectionDetails ConnectionDetails { get; private set; }
-        
+
         /// <summary>
-        /// The connection to the SQL database that commands will be run against.
+        /// A map containing all connections to the database that are associated with 
+        /// this ConnectionInfo's OwnerUri.
+        /// This is internal for testing access only
         /// </summary>
-        public DbConnection SqlConnection { get; set; }
+        internal readonly Dictionary<string, DbConnection> ConnectionTypeToConnectionMap =
```

Comment:
Ah, well, I guess that makes sense. It's too bad we don't have partial enums in C#. At least I'd hope we specify the strings as constants in the services that will be calling the connection service.

In that case, I'd say we don't really need a whole class for two constants. But I'll leave that up to you.